ui/src/main/webapp/src/app/reports/jpa/jpa-report.component.html

```diff
@@ -0,0 +1,157 @@
+<style>
+.panel-primary.wuTechnologies { border-color: #e7e7e7; }
+.panel-primary.wuTechnologies .panel-heading { background-color: white; padding: 15px 15px; border-bottom: 2px solid #e7e7e7; }
+.panel-primary.wuTechnologies .panel-heading .panel-title { color: black; font-size: 20px; font-weight: 500; }
+</style>
+
+<div class="panel-body">
+
+
+    <div class="header-bar">
+        <h2>Technologies &gt; <span style="color: #39a5dc;">JPA</span> </h2>
+        <div class="search-and-create">
+            <div class="search-and-create"  *ngIf="true">
+                <wu-search [(searchValue)]="searchText" (searchValueChange)="updateSearch()"></wu-search>
+            </div>
+        </div>
+    </div>
+
+
+    <div class="panel panel-default panel-primary wuTechnologies">
+        <div class="panel-heading">
+            <h3 class="panel-title">JPA Entities</h3>
+        </div>
+        <div class="panel-body">
```

Comment:
Wrapping `<div class="panel">` is already in `wu-table`, you don't have to repeat it